dataplane/internal/kernel/tap.go

```diff
@@ -27,6 +27,7 @@ func NewTap(name string) (*TapInterface, error) {
 			Name: name,
 		},
 		Mode:   netlink.TUNTAP_MODE_TAP,
+		Flags:  netlink.TUNTAP_MULTI_QUEUE_DEFAULTS,
```

Comment:
Why is this flag needed?

Reply:
these the "high performance" defaults per the documentation